v3/src/models/data/attribute.ts

```diff
@@ -192,10 +207,11 @@ export const Attribute = types.model("Attribute", {
     self.editable = editable
   },
   clearFormula() {
-    self.formula.setDisplayFormula("")
+    this.setDisplayFormula("")
   },
   setDisplayFormula(displayFormula: string) {
     self.formula.setDisplayFormula(displayFormula)
+    this.setEditable(displayFormula === "")
```

Comment:
The `editable` property is meant to correspond to the user's ability to specify editability. I think for this purpose a better solution would be to add a view like:
```
  get isEditable() {
    return self.editable && !self.displayFormula
  }
```